usage/dist/pom.xml

```diff
@@ -63,6 +63,35 @@
     <build>
         <plugins>
             <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <!-- copy the config file from the CLI project -->
+                <executions>
+                    <execution>
+                        <id>copy</id>
```

Comment:
Better use `unpack` goal with `includes` filter. This will make the bom artifact type on cli unnecessary.